### ISSUE TITLE
FTRL solver

### ISSUE THREAD

**Issue by binga** [scikit-learn-contrib/lightning#57]
Hello,

Are there any plans of including FTRL solver to this library? I guess it's fairly straight forward. I'm sorry I might not be able to contribute it myself because I am not fluent in it.
Reference: 
1. https://www.eecs.tufts.edu/~dsculley/papers/ad-click-prediction.pdf
2. https://www.kaggle.com/c/avazu-ctr-prediction/forums/t/10927/beat-the-benchmark-with-less-than-1mb-of-memory

Thanks!

**Comment by mblondel** [comment]
What's your motivation?

**Comment by binga** [comment]
For large datasets, we typically tend to use online learning and FTRL is known to do well because of the adaptive learning rate scheme and it's ability to activate a co-ordinate only after a category has been seen a good number of times in training data. In many competitions on Kaggle, I have seen FTRL doing far better than SGD and I believe it's a good algorithm to add to this library!

**Comment by mpearmain** [comment]
+1 for FTRL

**Comment by CasyWang** [comment]
+1 for FTRL

**Comment by gom7745** [comment]
+1 for FTRL

**Comment by sergei3000** [comment]
+1 for FTRL

**Comment by SimpleJian** [comment]
+1 for FTRL

**Comment by mblondel** [comment]
You can give as many +1 as you want, FTRL is not gonna implement itself :b

**Comment by DachuanZhao** [comment]
Any processing now ?
